coordinator/internal/orm/batch.go

```diff
@@ -31,6 +31,7 @@ type Batch struct {
 	WithdrawRoot    string `json:"withdraw_root" gorm:"column:withdraw_root"`
 	ParentBatchHash string `json:"parent_batch_hash" gorm:"column:parent_batch_hash"`
 	BatchHeader     []byte `json:"batch_header" gorm:"column:batch_header"`
+	EnableEncode    bool   `json:"enable_encode" gorm:"column:enable_encode"`
```

Comment:
can this be derived from version in `BatchHeader`?

base on the logic in function `getBatchTaskDetail`, if `EnableEncode=false`, you will use `codecv3.NewDABatch` and if `EnableEncode=true`, you will use `codecv4.NewDABatch(batchEncoding, true)`.